scripts/aws/destroy-old-clusters.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```

Comment:
Wouldn't this be much simpler if written in go? What makes `/bin/bash` the right choice here?

Reply:
cloudtest use bash commands to control aws cloud for testing. This script just a Wrapper of golang programm to use it with cloudtest